spiceaidocs/docs/components/models/anthropic.md

```diff
@@ -1,36 +1,30 @@
 ---
-title: 'Anthropic Language Models'
+title: 'Anthropic Models'
+description: 'Instructions for using language models hosted on Anthropic with Spice.'
 sidebar_label: 'Anthropic'
 sidebar_position: 5
 ---
 
-To use a language model hosted on Anthropic, specify `anthropic` in `from`.
+To use a language model hosted on Anthropic, specify `anthropic` in the `from` field.
 
-For a specific model, include it as the model ID in `from` (see example below). Defaults to `"claude-3-5-sonnet-20240620"`.
-These parameters are specific to Anthropic models:
+To use a specific model, include its model ID in the `from` field (see example below). If not specified, the default model is `"claude-3-5-sonnet-20240620"`.
```

Comment:
Default is now `"claude-3-5-sonnet-latest"`
 https://github.com/spiceai/spiceai/blob/cedefdb5b9e6860cdb3cd71303454f207fb122a7/crates/llms/src/anthropic/mod.rs#L38